curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/6750651aafcea04c6958a4e5.md

```diff
@@ -0,0 +1,88 @@
+---
+id: 6750651aafcea04c6958a4e5
+title: Task 40
+challengeType: 22
+dashedName: task-40
+---
+
+<!-- (Audio) James: Everything else was running smoothly once the database issue was fixed. -->
+
+# --instructions--
+
+Listen to the audio and complete the sentence below.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Everything else BLANK running smoothly once the database issue was fixed.`
+
+## --blanks--
+
+`was`
+
+### --feedback--
+
+This word is the singular form of the verb `to be` in the Simple Past, used with subjects like `everything`, `something`, `anything`, and `nothing`.
+
+# --explanation--
+
+Words like `everything`, `something`, `anything`, and `nothing` are indefinite pronouns that take singular verbs, even though they may refer to multiple items or concepts. Here are examples for each:  
+
+- `Everything was ready for the presentation.` (`everything` refers to all things and requires a singular verb)
+
+- `Something is wrong with the code.` (`something` refers to an unspecified thing and takes a singular verb)
+
+- `Is anything missing from the report?` (`anything` refers to any one thing in a general sense and uses a singular verb)
+
+- `Nothing was found in the logs.`  (`nothing` refers to the absence of things and requires a singular verb)
+
+In the sentence `Everything else was running smoothly`, you use `was` because `everything` is treated as singular, even though it refers to multiple elements.
```

Comment:
```suggestion
Words like `everything`, `something`, `anything`, and `nothing` are indefinite pronouns that take singular verbs, even though they may refer to multiple items or concepts. For example:  

- `Everything was ready for the presentation.` - `Everything` refers to all things and requires a singular verb.

- `Something is wrong with the code.` - `Something` refers to an unspecified thing and takes a singular verb.

- `Is anything missing from the report?` - `Anything` refers to any one thing in a general sense and uses a singular verb.

- `Nothing was found in the logs.`  - `Nothing` refers to the absence of things and requires a singular verb)

In the sentence `Everything else was running smoothly`, you use `was` because `everything` is treated as singular, even though it refers to multiple elements.
```